src/test/java/edu/kis/vh/stacks/unittests/StackTest.java

```diff
@@ -7,6 +7,9 @@
 
 public class StackTest {
 
+	private static final int STACK_CAPACITY = 11;
```

Comment:
Zmiana dotyczyła: "Zmiana zwracanej wartości gdy stos pusty". Nie powinno to mieć związku ze zmianą pojemności stosu. Takie "poprawki" testów są nie dopuszczalne. Trzeba się "zatrzymać" i pomyśleć!